contents/handbook/engineering/feature-ownership.md

```diff
@@ -15,35 +15,38 @@ You can also view the list [directly in GitHub](https://github.com/PostHog/posth
-| Recordings | [@rcmarron][@rcmarron]  |  <span class="lemon-tag gh-tag">recordings</span> |
-| Retention | [@EDsCODE][@EDsCODE]  |  <span class="lemon-tag gh-tag">retention</span> |
-| Saved Insights | [@Twixes][@Twixes]  |  <span class="lemon-tag gh-tag">saved-insights</span> |
-| Session Analytics | [@rcmarron][@rcmarron]  |  <span class="lemon-tag gh-tag">sessions</span> |
-| Settings (personal & project) | [@liyiy][@liyiy]  |  <span class="lemon-tag gh-tag">settings</span> |
-| Stickiness | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">stickiness</span>  |
-| Taxonomy | [@alexkim205][@alexkim205]  | <span class="lemon-tag gh-tag">taxonomy</span>  |
-| Toolbar | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">toolbar</span>  |
-| Trends | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">trends</span>  |
+| Actions | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">feature/actions</span> |
+| Actors Modal | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">feature/actors-modal</span>  |
+| Annotations | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">feature/annotations</span> |
+| API Structure | [@Twixes][@Twixes]  | <span class="lemon-tag gh-tag">feature/api-structure</span>  |
+| Application Performance Monitoring (APM) | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">feature/apm</span>  |
+| Async migrations | [@yakkomajuri][@yakkomajuri]  | <span class="lemon-tag gh-tag">feature/async-migrations</span> |
+| Billing | [@timgl][@timgl]  |  <span class="lemon-tag gh-tag">feature/billing</span> |
+| Cohorts | [@EDsCODE][@EDsCODE]  |  <span class="lemon-tag gh-tag">feature/cohorts</span>  |
+| Correlation Analysis | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">feature/correlation-analysis</span> |
+| Dashboards | [@Twixes][@Twixes]  |  <span class="lemon-tag gh-tag">feature/dashboards</span> |
+| Events | [@alexkim205][@alexkim205]  |  <span class="lemon-tag gh-tag">feature/events</span>  |
+| Experimentation | [@neilkakkar][@neilkakkar] |  <span class="lemon-tag gh-tag">feature/experimentation</span> |
+| Feature Flags | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">feature/feature-flags</span> |
+| Funnels | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">feature/funnels</span>  |
+| Group Analytics | [@macobo][@macobo]  |  <span class="lemon-tag gh-tag">feature/group-analytics</span> |
+| Onboarding | [@liyiy][@liyiy]  | <span class="lemon-tag gh-tag">feature/onboarding</span>  |
+| Lifecycle | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">feature/lifecycle</span>  |
+| Paths | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">feature/paths</span> |
+| Permissions | [@Twixes][@Twixes]  | <span class="lemon-tag gh-tag">feature/permissions</span>  |
+| Persons | [@macobo][@macobo]  | <span class="lemon-tag gh-tag">feature/persons</span>  |
```

Comment:
Can't remember what was the conclusion on who should own this. Was it @alexkim205 ? CC @marcushyett-ph & @macobo too

Reply:
I think it was @alexkim205 ?

Reply:
:+1: for not me at least for the next few months :joy:

Reply:
Hm, I don't remember being the owner for this 😅 but glad to take it on!

Reply:
```suggestion
| Persons | [@alexkim205][@alexkim205]  | <span class="lemon-tag gh-tag">feature/persons</span>  |
```